public/video-ui/src/components/ManagedForm/ManagedForm.js

```diff
@@ -6,7 +6,7 @@ import { ManagedSection } from './ManagedSection';
 export class ManagedForm extends React.Component {
   static propTypes = {
     data: PropTypes.object,
-    updateData: PropTypes.func.isRequired,
+    updateData: PropTypes.func,
```

Comment:
Is this still left over?

Reply:
Should have added a comment here, sorry.

This is a necessary change. We're using `ManagedForm` for the CCD component, however the CCD component is completely read-only, so doesn't need to specify an `updateData` function... even if it did, it would be `() => undefined` which is pointless. Much nicer to not make `updateData` required.

Reply:
Obviously we could not use `ManagedForm` for this component, but I cba to work out the styling atm 😄 .